packages/frontend/component/src/components/block-suite-editor/index.tsx

```diff
@@ -77,6 +74,52 @@ const useBlockElementById = (
   return blockElement;
 };
 
+/**
+ * TODO: Define error to unexpected state together in the future.
+ */
+export class NoPageRootError extends Error {
+  constructor(public page: Page) {
+    super('Page root not found when render editor!');
+
+    // Log info to let sentry collect more message
+    const spaceVectors = Array.from(page.doc.spaces.entries()).map(
+      ([pageId, doc]) => `${pageId} > ${doc.guid}`
+    );
+    console.info(
+      'NoPageRootError current data: %s',
+      JSON.stringify({
+        expectPageId: page.id,
+        expectGuid: page.spaceDoc.guid,
+        spaceVectors,
+      })
+    );
+  }
+}
+
+const PAGE_ROOT_KEY = Symbol('PAGE_ROOT');
+function loadPageRoot(page: Page) {
```

Comment:
this is a react hook function, should be named as `useLoadPageRoot`

Reply:
it is not a hook, it is a suspense load function.